parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -340,7 +340,6 @@ removeContainerImage() {
 cleanUpImages() {
     local targetImage=$1
     export targetImage
-    export -f removeContainerImage
```

Comment:
looks like removeContainerImage is needed here since it'll be used in retrycmd_if_failure?

Reply:
I've moved it to cleanUpContainerImages where all the other exports are for clarity...testing if that works

Reply:
right that would export for cleanUpImages. but I believe when cleanUpImages calls retrycmd_if_failure, it has to be exported again. lmk if that's not true.

Reply:
I tested locally, cleanUpImages runs after the export so it still works.